src/main/java/com/knu/buga1chuk/constant/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.knu.buga1chuk.constant;
+
+public final class Constants {
+    private Constants() {
+    }
+
+    public static final String PERSON_FORMAT = "%d,%s,%d,%s";
```

Comment:
Move this constant into the `Person` model

Reply:
fix